mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -68,7 +68,6 @@ import { IStreamlitPreviewManager } from '../AppPreview/StreamlitPreviewPlugin';
 import { waitForNotebookReady } from '../../utils/waitForNotebookReady';
 import { getBase64EncodedCellOutputInNotebook } from './utils';
 import { logEvent } from '../../restAPI/RestAPI';
-import { isUserSignedUp } from '../../utils/userSignupState';
```

Comment:
Should we move the cod from userSignupState into the hook file? It would help us make sure we always use the hook I guess, which could be nice